ui/app/utils/properties/style-string.js

```diff
@@ -8,7 +8,7 @@ import { htmlSafe } from '@ember/template';
 //     styleStr: styleStringProperty('styleProps') // color:#FF0;border-width:1px
 export default function styleStringProperty(prop) {
   return computed(prop, function() {
-    const styles = this.get(prop);
+    const styles = get(this, prop);
```

Comment:
I briefly tried using this in the DAS realm and it failed, maybe fixing it was this simple??? haha oops 🙃